misc/.gitignore

```diff
@@ -0,0 +1,2 @@
+exec
+exec.json
```

Comment:
Should we merge all nested .gitignore files into one file in the root directory?